src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -206,6 +234,27 @@ public bool IsBlocked(IReadOnlyStore snapshot, UInt160 account)
             return snapshot.Contains(CreateStorageKey(Prefix_BlockedAccount, account));
         }
 
+        /// <summary>
+        /// Sets the block generation time in milliseconds.
+        /// </summary>
+        /// <param name="engine">The execution engine.</param>
+        /// <param name="value">The block generation time in milliseconds. Must be between 1 and MaxBlockGenTime.</param>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the provided value is outside the allowed range.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the caller is not a committee member.</exception>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.States | CallFlags.AllowNotify)]
+        public void SetMSPerBlock(ApplicationEngine engine, uint value)
+        {
+            if (value == 0 || value > MaxMSPerBlock)
```

Comment:
```suggestion
            if (value <= 0 || value > MaxMSPerBlock)
```

Reply:
It’s uint, not needed. The same code is implemented for other setters.

Reply:
I didn't saw uint, my bad